### ISSUE TITLE
Passing --svg-dimattrs results in merged icons

### ISSUE THREAD

**Issue by tomekbuszewski** [jkphl/svg-sprite#138]
Hi,

first of all - great tool ;-)

But I found a problem. I use cli for my needs, this is my code:
`svg-sprite --css-render-scss-dest _sprite.scss --css-example -cD . --cscss _img/*.svg`
It works great. But when I add `--svg-dimattrs=false`, every generated icon looks the same (only scaled).

**Comment by jkphl** [comment]
Thanks for reporting this @tomekbuszewski, I'll have a look into it as soon as I find some time!

**Comment by Johnius** [comment]
We are just need to add `background-size` with svg dims to each css class.

**Comment by jkphl** [comment]
@Johnius In how far does your need correspond to @tomekbuszewski's one? As far as I understood him, he's just not able to *suppress* the output of dimensions using the CLI tool.

**Comment by tomekbuszewski** [comment]
I've made a screenshot of how it looks on preview, when I add `--svg-dimattrs=false`: https://www.dropbox.com/s/pguyz7lb3sufhb5/Screenshot%202016-02-08%2012.12.40.png?dl=0

Perhaps I just didn't understand some commands and/or I am using them in wrong combination. But this doesn't seem like a proper output.

**Comment by Johnius** [comment]
@jkphl As I understood he wants to disable explicit setting of sprite dimensions in resulting svg. But when he do that it raises wrong results in browser — as he mentioned above in screenshot. I got the same results. After some digging it seems that we need `background-size` property on every css class containing our sprite in `background` property. It will fix @tomekbuszewski problem.

**Comment by jkphl** [comment]
OK guys, I think I finally got the problem. ;)

What seems to be buggy: The example document should either ignore that the dimension attributes have been disabled (i.e. a dimension attribute enabled sprite should be used for the example document), or the background size should be set as described.

In general, however, the result of `--svg-dimattrs=false` looks correct to me, and so does the browser behavior. So it's more the example document I've got to fix.
